flexget/plugins/list/series_list.py

```diff
@@ -0,0 +1,438 @@
+    ids = relationship('SeriesListID', backref='series', cascade='all, delete, delete-orphan')
+
+    def __init__(self, title):
+        self.title = title
+
+    def __repr__(self):
+        return '<SeriesListSeries title=%s,list_id=%d>' % (self.title, self.list_id)
+
+    def format_converter(self, attribute):
+        """
+        Return correct attribute format, based on schema. This is needed to maintain consistency with multi
+        type schema formats, and convert instrumented lists on the fly
+        """
+        value = getattr(self, attribute)
+        if not value:
+            return
+        if isinstance(value, InstrumentedList):
+            if attribute.endswith('regexp'):
+                return [regexp.regexp for regexp in value]
+            elif attribute == 'alternate_name':
+                return [alternate_name.name for alternate_name in value]
+            elif attribute == 'qualities':
+                return [quality.quality for quality in value]
+            elif attribute == 'from_group':
+                return [group.group_name for group in value]
+            elif attribute == 'special_ids':
+                return [special_id.id_name for special_id in value]
+        if attribute in ['propers', 'tracking']:
+            # Value can be either bool or unicode
+            if value in ['0', '1']:
+                return bool(value)
+        if attribute == 'quality':
+            return value.quality
+        return value
+
+    def to_entry(self):
+        entry = Entry()
+        entry['title'] = entry['series_name'] = self.title
+        entry['url'] = 'mock://localhost/series_list/%d' % self.id
+        entry['set'] = {}
+
+        for attribute in SERIES_ATTRIBUTES:
+            # `set` is not a real series attribute and is just a way to pass IDs in task.
+            if attribute == 'set':
+                continue
+            result = self.format_converter(attribute)
+            if result:
+                # Maintain support for configure_series plugin expected format
+                entry['configure_series_' + attribute] = entry[attribute] = result
+        for series_list_id in self.ids:
+            entry[series_list_id.id_name] = series_list_id.id_value
+            entry['set'].update({series_list_id.id_name: series_list_id.id_value})
+        return entry
+
+    def to_dict(self):
+        series_dict = {
+            'id': self.id,
+            'added_on': self.added,
+            'title': self.title,
+            'list_id': self.list_id,
+            'series_list_ids': [series_list_id.to_dict() for series_list_id in self.ids]
+        }
+        for attribute in SETTINGS_SCHEMA['properties']:
+            # `set` is not a real series attribute and is just a way to pass IDs in task.
+            if attribute == 'set':
+                continue
+            series_dict[attribute] = self.format_converter(attribute)
+        return series_dict
+
+
+class SeriesListID(Base):
```

Comment:
I would call this SeriesListSeriesExternalID because that's what this is .. or at least describe a bit in a docstring